pkg/util/execdetails/execdetails.go

```diff
@@ -925,18 +969,123 @@ func (context *TiFlashScanContext) Clone() TiFlashScanContext {
+		totalDeltaBytes:                     context.totalDeltaBytes,
+		totalMvccInputRows:                  context.totalMvccInputRows,
+		totalMvccInputBytes:                 context.totalMvccInputBytes,
+		totalMvccOutputRows:                 context.totalMvccOutputRows,
+		totalLateMaterializationSkipRows:    context.totalLateMaterializationSkipRows,
+		totalBuildBitmapTimeMs:              context.totalBuildBitmapTimeMs,
+		totalBuildInputStreamTimeMs:         context.totalBuildInputStreamTimeMs,
+		numStaleRead:                        context.numStaleRead,
+		localMinStreamCostMs:                context.localMinStreamCostMs,
+		localMaxStreamCostMs:                context.localMaxStreamCostMs,
+		remoteMinStreamCostMs:               context.remoteMinStreamCostMs,
+		remoteMaxStreamCostMs:               context.remoteMaxStreamCostMs,
+		regionNumOfInstance:                 make(map[string]uint64),
+	}
+	for k, v := range context.regionNumOfInstance {
+		newContext.regionNumOfInstance[k] = v
+	}
+	return newContext
 }
+
 func (context *TiFlashScanContext) String() string {
-	return fmt.Sprintf("tiflash_scan:{dtfile:{total_scanned_packs:%d, total_skipped_packs:%d, total_scanned_rows:%d, total_skipped_rows:%d, total_rs_index_check_time: %dms, total_read_time: %dms, total_disagg_read_cache_hit_size: %d, total_disagg_read_cache_miss_size: %d}, total_create_snapshot_time: %dms, total_local_region_num: %d, total_remote_region_num: %d, total_learner_read_time: %dms}", context.totalDmfileScannedPacks, context.totalDmfileSkippedPacks, context.totalDmfileScannedRows, context.totalDmfileSkippedRows, context.totalDmfileRoughSetIndexCheckTimeMs, context.totalDmfileReadTimeMs, context.totalDisaggReadCacheHitSize, context.totalDisaggReadCacheMissSize, context.totalCreateSnapshotTimeMs, context.totalLocalRegionNum, context.totalRemoteRegionNum, context.totalLearnerReadMs)
+	regionBalanceInfo := "none"
+	if len(context.regionNumOfInstance) > 0 {
+		maxNum := uint64(0)
+		minNum := uint64(math.MaxUint64)
+		for _, v := range context.regionNumOfInstance {
+			if v > maxNum {
+				maxNum = v
+			}
+			if v > 0 && v < minNum {
+				minNum = v
+			}
+		}
+		regionBalanceInfo = fmt.Sprintf("{instance_num: %d, max/min: %d/%d=%f}",
+			len(context.regionNumOfInstance),
+			maxNum,
+			minNum,
+			float64(maxNum)/float64(minNum))
+	}
+	dmfile_disagg_info := ""
+	if context.totalDisaggReadCacheHitSize != 0 || context.totalDisaggReadCacheMissSize != 0 {
+		dmfile_disagg_info = fmt.Sprintf(", disagg_cache_hit_size: %d, disagg_cache_miss_size: %d",
+			context.totalDisaggReadCacheHitSize,
+			context.totalDisaggReadCacheMissSize)
+	}
+	return fmt.Sprintf("tiflash_scan:{"+
+		"dtfile:{"+
+		"data_scanned_rows:%d, "+
+		"data_skipped_rows:%d, "+
+		"mvcc_scanned_rows:%d, "+
+		"mvcc_skipped_rows:%d, "+
+		"lm_filter_scanned_rows:%d, "+
+		"lm_filter_skipped_rows:%d, "+
+		"rs_index_check_time: %dms, "+
+		"read_time: %dms"+
```

Comment:
```suggestion
		"total_rs_index_check:%dms, "+
		"total_read:%dms"+
```

I added a `total_` for these time metrics, so that we will know it is a total, not a avg, min or max.

BTW I removed `_time`, because we know it is a time from the `ms` unit.